src/editor/components/ControlPanel/ControlPanel.tsx

```diff
@@ -26,35 +26,52 @@ const ControlPanelViewModeIcons = {
 export interface ControlPanelProps extends ClassNameProps {
     viewMode?: ViewModeItem;
     onViewModeChange: (viewMode: ViewModeItem) => void;
+    theme: Theme;
+    onThemeSwitch: (theme: Theme) => void;
 }
 
 const ControlPanel = ({
     viewMode = ViewModeItem.Edititng,
     onViewModeChange,
     className,
-}: ControlPanelProps) => (
-    <div className={b(null, className)}>
-        <div className={b('mode-switch')}>
-            <span>{i18n('mode')}</span>
-            <RadioButton
-                className={b('radio-button')}
-                value={viewMode}
-                onUpdate={(value) => onViewModeChange(value as ViewModeItem)}
-            >
-                {Object.values(ViewModeItem).map((item) => {
-                    const Icon = ControlPanelViewModeIcons[item];
-
-                    return (
-                        <RadioButton.Option key={item} value={item}>
-                            <span className={b('icon')}>
-                                <Icon width={ICON_SIZE} height={ICON_SIZE} />
-                            </span>
-                        </RadioButton.Option>
-                    );
-                })}
-            </RadioButton>
+    theme,
+    onThemeSwitch,
+}: ControlPanelProps) => {
+    return (
+        <div className={b(null, className)}>
+            <div></div>
+            <div className={b('mode-switch')}>
+                <span>{i18n('mode')}</span>
+                <RadioButton
+                    className={b('radio-button')}
+                    value={viewMode}
+                    onUpdate={(value) => onViewModeChange(value as ViewModeItem)}
+                >
+                    {Object.values(ViewModeItem).map((item) => {
+                        const Icon = ControlPanelViewModeIcons[item];
+
+                        return (
+                            <RadioButton.Option key={item} value={item}>
+                                <span className={b('icon')}>
+                                    <Icon width={ICON_SIZE} height={ICON_SIZE} />
+                                </span>
+                            </RadioButton.Option>
+                        );
+                    })}
+                </RadioButton>
+            </div>
+            <div className={b('theme-switch')}>
+                <span>{i18n('Theme')}</span>
+                <Select value={[theme]} onUpdate={(value) => onThemeSwitch(value[0] as Theme)}>
```

Comment:
lets use same naming like `onViewModeChange` and `onThemeChange` or `onViewModSwitch` and `onThemeSwitch`

Reply:
renamed to `onThemeChange`